README.md

```diff
@@ -3,6 +3,7 @@
 |Branch|Status|
 |---|---|
 |dev|[![Build Status](https://dev.azure.com/azfunc/Azure%20Functions/_apis/build/status/Azure.azure-functions-host?branchName=dev)](https://dev.azure.com/azfunc/Azure%20Functions/_build/latest?definitionId=37&branchName=dev)|
+|release/4.x|[![Build Status](https://dev.azure.com/azfunc/Azure%20Functions/_apis/build/status/Azure.azure-functions-host?branchName=release%2F3.0)](https://dev.azure.com/azfunc/Azure%20Functions/_build/latest?definitionId=37&branchName=release%2F4.x)|
```

Comment:
It looks like this links to release/3.0 in the first URI, but release/4.x in the second; these should match.